### PR TITLE
Web3 E2E Mock

### PR DESCRIPTION
**Pull request by FrederikBolding**
Add web3 mock for e2e testing such that we can remove private keys entirely from the codebase.

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7216]

File: src/utils/ethereumMock.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { ethers, Wallet } from 'ethers';
```
Comment (vilsbole) [inline src/utils/ethereumMock.ts:1]: 1. I understand that you forked a project and adapted it to our repo.
Please include an explanation and source links in a comment to share the research. 
2. This is only used in tests. Please move out of `src/utils`. We can place it in `__tests__`?

**Review by vilsbole — CHANGES_REQUESTED** [review]
Nice work!

File: src/utils/ethereumMock.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/utils/ethereumMock.ts:1]: Jujst to clarify, there was no fork done for this PR. It was for the other experiment I did with Web3 E2E, which was dropped. Will move the file.

**Review by vilsbole — APPROVED** [review]

**Pull request merged**